<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Library</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="Pages">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>56</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="signin_page">
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>50</y>
          <width>201</width>
          <height>71</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic;&quot;&gt;Welcome&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic;&quot;&gt;to your Library :)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>160</y>
          <width>221</width>
          <height>231</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLineEdit" name="lineEdit_user">
           <property name="placeholderText">
            <string>username</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_pass_2">
           <property name="placeholderText">
            <string>password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Sign In</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>400</y>
          <width>31</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>or...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_signup">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>430</y>
          <width>100</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Sign Up</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="db_status">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>510</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="books_page">
       <widget class="QListWidget" name="list_books">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>751</width>
          <height>461</height>
         </rect>
        </property>
       </widget>
       <widget class="QCommandLinkButton" name="commandLinkButton_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>480</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Log out</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="book_admin">
       <widget class="QListWidget" name="admin_List">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>50</y>
          <width>581</width>
          <height>421</height>
         </rect>
        </property>
       </widget>
       <widget class="QCommandLinkButton" name="commandLinkButton_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Go Back</string>
        </property>
       </widget>
       <widget class="QPushButton" name="add_bookButton">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>280</y>
          <width>71</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>161</width>
          <height>166</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="title_Edit">
           <property name="placeholderText">
            <string>title</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="authorEdit">
           <property name="placeholderText">
            <string>author</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="aboutEdit">
           <property name="placeholderText">
            <string>about</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="picdirEdit">
           <property name="placeholderText">
            <string>picture directory</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="refresh_book">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>480</y>
          <width>100</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="admin_page">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>70</y>
          <width>261</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu Mono</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Welcome to the admin menu</string>
        </property>
       </widget>
       <widget class="QLabel" name="users_icon">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>200</y>
          <width>161</width>
          <height>151</height>
         </rect>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="pixmap">
         <pixmap>group+people+team+users+icon-1320196240876938595_256.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="books_icon">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>200</y>
          <width>151</width>
          <height>151</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>icon-book.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_users">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>390</y>
          <width>151</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>User Managment</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_books">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>390</y>
          <width>161</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Book Managment</string>
        </property>
       </widget>
       <widget class="QCommandLinkButton" name="commandLinkButton_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>470</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Log out</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="users_managment">
       <widget class="QPushButton" name="add_userButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>260</y>
          <width>71</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QPushButton" name="delete_userButton">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>260</y>
          <width>71</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QPushButton" name="update_userButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>300</y>
          <width>71</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
       <widget class="QCommandLinkButton" name="commandLinkButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>196</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Go Back</string>
        </property>
       </widget>
       <widget class="QTableView" name="dbtableView">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>50</y>
          <width>473</width>
          <height>381</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>181</width>
          <height>166</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="man_stu_noEdit">
           <property name="placeholderText">
            <string>student number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="man_usernameEdit">
           <property name="placeholderText">
            <string>username(for delete)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="man_passwordEdit">
           <property name="placeholderText">
            <string>password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="man_is_adminEdit">
           <property name="placeholderText">
            <string>is_admin</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLineEdit" name="rowidlineEdit">
        <property name="geometry">
         <rect>
          <x>8</x>
          <y>300</y>
          <width>151</width>
          <height>28</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>rowid(for update)</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="signup_page">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>50</y>
          <width>201</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Please Fill Out This From:</string>
        </property>
       </widget>
       <widget class="QCommandLinkButton" name="commandLinkButton_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>196</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Go Back</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>110</y>
          <width>221</width>
          <height>321</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="sign_up_stuno">
           <property name="placeholderText">
            <string>Student number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sign_up_username">
           <property name="placeholderText">
            <string>Username</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sign_up_pass">
           <property name="placeholderText">
            <string>Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sign_u_confirm">
           <property name="placeholderText">
            <string>Confrim Password</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="signUPButton">
           <property name="text">
            <string>Sign Up</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
